TodoApp.ConsoleApp/Services/TodoService.cs

```diff
@@ -1,77 +1,87 @@
 ﻿using System.Collections.Generic;
-using TodoApp.Library.Data;
-using TodoApp.Library.Models;
+using TodoApp.ConsoleApp.Repositories.Models;
+using TodoApp.ConsoleApp.Repositories.XMLRepository;
 using System.Linq;
 using System;
+using System.IO;
+using System.Xml;
+using System.Text;
 
 namespace TodoApp.ConsoleApp.Services
 {
     public class TodoService : ITodoService
     {
-        public IEnumerable<Task> GetAll()
-        {
-            XMLTaskReader reader = new XMLTaskReader();
-
-            IEnumerable<Task> tasks = reader.ReadTasks();
+        static string Path { get; set; }
+        TodoRepository repo = new TodoRepository(Path);
```

Comment:
When initializing the repo, `Path` is null. It is not set in `TaskOperations`.
Please, create a constructor for `TodoService` to correctly initialize it with the repo.

Reply:
Make sure that if the path provided is valid, but there is no file created at that destination, we will create it.